src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -592,7 +596,7 @@ private static byte percentDecode(final byte[] bytes, final int start) {
         return ((byte) ((c1 << 4) + c2));
     }
 
-    public static String percentDecode(final String source) {
+    private static String percentDecode(final String source) {
```

Comment:
I assumed we want these private. All public methods/constants/etc. should have javadoc now.

Reply:
Yes, if it was not `public` in 1.5.0, it shouldn't be now.